watergun_assassin/authentication/tests.py

```diff
@@ -0,0 +1,54 @@
+from django.test import TestCase
+from django.urls import reverse
+from datetime import datetime
+
+
+class UserCreationTest(TestCase):
+    def test_valid_payload(self):
```

Comment:
If you check out [the Writing Tests section of the Django docs](https://docs.djangoproject.com/en/3.0/topics/testing/overview/#writing-tests), in the example you'll notice that underneath the function definition there is a doc (`"""`) comment that describes what the test is testing. Let's add a description for each of the tests in this file.